<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>10</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>No user</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>129</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="AutoBut">
       <property name="text">
        <string>Авторизация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreaBut">
       <property name="text">
        <string>Создать Заказ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CurrBut">
       <property name="text">
        <string>Текущие Заказы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HistBut">
       <property name="text">
        <string>История Заказов</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SuppBut">
       <property name="text">
        <string>Тех. Поддержка</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExitBut">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
